### ISSUE TITLE
Version 3.6 |  Email is not sending automatically 

### ISSUE THREAD

**Issue by whitesparrowbd** [iNextrix/ASTPP#436]
I have tried many times in many way 
E-mail on signup, forgot password or anyone is not sending automatically, i need to send it from manually from E-mail history page.

I have already enable E-mail notification from Configuration > Settings

Can you help me to solve it?

**Comment by smrdoshi** [comment]
Have you configured crons?

**Comment by whitesparrowbd** [comment]
yes configured the given crons 

# Generate Invoice  
0 1 * * * cd /var/www/html/astpp/cron/ && php cron.php GenerateInvoice 
# Low balance notification
0 1 * * * cd /var/www/html/astpp/cron/ && php cron.php UpdateBalance          
 # Low balance notification
0 0 * * * cd /var/www/html/astpp/cron/ && php cron.php LowBalance 
# Low credit notification
0 0 * * * cd /var/www/html/astpp/cron/ && php cron.php LowCredit          
 # Update currency rate
0 0 * * * cd /var/www/html/astpp/cron/ && php cron.php CurrencyUpdate
--

i saved all data to /etc/crontab

but the main issue is all the given crons are regular usable issue, but where is the signup, forgot password, confirmation and others cron job ?

**Comment by devangn** [comment]
About 'email not sending issue', you can enable mail debug from configuration and monitor the logs in /var/log/astpp/astpp_email.log file.
About other event's cron, those event are triggered from button click and managed by calling according scripts when required/button clicked.

**Comment by powerpbx** [comment]
Maybe a stupid question but have you verified that emails sent manually work?

`echo testing | mail -s "test mail" somemail@somedomain.com`

**Comment by AlexHeylin** [comment]
@whitesparrowbd I think you've made a mistake I made when I first configured ASTPP by hand.  You've put the cron entries in /etc/crontab directly - but they're in the wrong format to go directly into the master crontab as it needs a username field too so they won't run.  I suggest moving them to /var/spool/cron/crontabs/astpp on Debian or /var/spool/cron/astpp on CentOS - or use the install.sh to install.  The cron code is on lines 556 - 576.  Using the install.sh will ensure the system is configured as it's supposed to be, as there are some other things like this that can catch you out if you try to install manually. 
@powerpbx makes a good suggestion too - confirm email system is working, and your settings in Configuration > Settings > Email are correct.
